README update

cr7st74n opened the pull request:
Revire the new README file. Thank you

cr7st74n committed:
README update

RichardJThies committed:
Update README.md

Fixed some minor mistakes, added dotenv

RichardJThies reviewed (approved):
Much improved!
fix #56

JSirisavath committed:
Update README.md

Updated the read me to let users know the setup process before running the app.
